### PR TITLE
Ignore coverage on some branches and fix e2e tests

### PR DESCRIPTION
**Pull request by limdauto**
## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [ ] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: package/features/steps/cli_steps.py [hunk 1]
```diff
@@ -80,6 +80,29 @@ def create_project_from_config_file(context):
     assert res.returncode == OK_EXIT_CODE
 
 
+@given("I have run a non-interactive kedro new with {starter} starter")
```
Comment (lorenabalan) [inline package/features/steps/cli_steps.py:83]: Does the starter name really matter or can we just say `with starter`? I think that's what we went with on kedro & kedro-docker.

**Review by MerelTheisenQB — APPROVED** [review]
LGTM!

**Comment by lorenabalan** [comment]
Forgot to approve but you're good. 😂

File: package/features/steps/cli_steps.py (hunk 1 above)
Comment (limdauto) [reply package/features/steps/cli_steps.py:83]: Yea it doesn't really matter. Funny that was what caused problems for me... I used "pandas-iris" in the test and it didn't work and I couldn't figure out why 🤦  should have just hard coded.

**Pull request merged**